crates/goose-mcp/src/developer/rmcp_developer.rs

```diff
@@ -187,6 +189,10 @@ impl ServerHandler for DeveloperServer {
 
                 Use the shell tool as needed to locate files or interact with the project.
 
+                Use the `analyze` tool aggressively to explore directory structures and understand codebases.
+                Leverage `analyze` through `return_last_only=true` subagents for deep codebase understanding with lean context
```

Comment:
should it just have this second line, more compressed, and it will know from the tool desc how to use it? (just thinking in terms of token efficiency here)